docs/tutorials/aws.md

```diff
@@ -1108,3 +1108,15 @@ args:
     --aws-zone-tags=team=k8s,vertical=platform # this is not supported
     --aws-zone-tags==tag-value # this is not supported
 ```
+
+### Add Roles specific to the zone
+
+If you have multiple zones and want to manage them with different roles, you can configure `external-dns` with the following option:
+
+```sh
+args:
+  --aws-domain-roles=example.com=arn:aws:iam::123456789012:role/external-dns-role
```

Comment:
Would you please complete this example with a second role on a second domain ?

Reply:
yup done